woof-code/rootfs-skeleton/usr/sbin/fontmanager

```diff
@@ -216,6 +283,24 @@ case $EXIT in
 	</selectfont> 
 </fontconfig>
 _EOF
+		 
+		 fc-cache -fv
+		 
+		 xrdb -merge $HOME/.Xresources
+
+		 gsettings set org.gnome.desktop.interface font-antialiasing ${gnomeantialias} 2>/dev/null		 	
```

Comment:
This will only affect applications running as root, not applications running as spot